user_manual/files/quota.rst

```diff
@@ -2,9 +2,8 @@
 Storage quota
 =============
 
-Your Nextcloud admin has the option to set a storage quota on users. Look at 
-the top of your Personal page to see what your quota is, and how much you have 
-used.
+Your Nextcloud admin has the option to set a storage quota on users. Look at
+of your Personal page to see what your quota is, and how much you have used.
```

Comment:
@Valdnet I'd say you would at least have to also remove "of" else the sentence is incorrect/incomplete, no?